docs/codes-other-values/glossary.rst

```diff
@@ -59,6 +59,21 @@ Glossary
 
    Universe
 
-    A type of job, describing some of the services it may need on an EP.  The default universe, with the
-    minimal additional services needed on the EP is called "vanilla".  Other universes are container, grid,
-    Grid, and VM.
+    A type of job, describing some of the services it may need on an EP.  The
+    default universe, with the minimal additional services needed on the EP is
+    called "vanilla".  Other universes include Container, Grid, and VM.
```

Comment:
```suggestion
    A type of job, describing some of the services it may need on an EP.  The
    default universe, with the minimal additional services needed, is
    called "vanilla".  Other universes include Container, Grid, and VM.
```